FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m

```diff
@@ -0,0 +1,34 @@
+//
+//  OguryFSTRPartnerInitializer.m
+//  FreestarAds-Ogury
+//
+//  Created by Carlos Alcala on 7/04/22.
+//
+
+#import "OguryFSTRPartnerInitializer.h"
+#import <OgurySdk/Ogury.h>
+#import <OguryAds/OguryAds.h>
+#import <OguryChoiceManager/OguryChoiceManager.h>
+
+@implementation OguryFSTRPartnerInitializer
+
++ (instancetype)shared {
+    static OguryFSTRPartnerInitializer *instance = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        instance = [[OguryFSTRPartnerInitializer alloc] init];
+    });
+    return instance;
+}
+
+- (void)runInitialization {
+    // run partner initialization here
+    FSTRLog(@"OGURY: Partner initialization.");
+
+    OguryConfigurationBuilder *configurationBuilder = [[OguryConfigurationBuilder alloc] initWithAssetKey:@"OGY-374D2BA758F4"];
```

Comment:
@carlosalcala-freestar are you sure you want hardcode the asset key?  do we hardcode asset keys in Android?

Reply:
@maddapper do we have the Ogury Asset Key in one of the Mediator/Adapter class properties ? In Yahoo we use the app_id but not sure in this case 🤔